packages/roosterjs-editor-types/lib/enum/KnownAnnounceStrings.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Known announce strings
+ */
+export enum KnownAnnounceStrings {
```

Comment:
Regular enum will cause additional code which can increase bundle size. We can use const enum, the our build code will general Compatible* type which is not const so that other people who don't want to use const enum can use it.

Then for all those places where this enum is used, also let them accept the Compatible* enum type and do "import type". 

You can search const enum for examples.